docs/guide/controller-programming.md

```diff
@@ -452,6 +452,19 @@ if (finished) {
 
 Note that the exit status as well as the value returned by the `main` function are ignored by Webots.
 
+### Console Output
+
+As mentioned earlier, printing to `stdout` or `stderr` from a controller will be redirected by default to the Webots console.
+Note however, that the Webots console doesn't support `stdin` input.
+Like most terminals, it supports a few basic [ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code) for setting text styles and clearing the content of the console:
+  - 3 bit color (foreground and background)
+  - Bold style
+  - Underline style
+  - Clear screen (same as issuing `clear` command in your terminal)
+  - Reset (colors and styles)
+
+There is a world example in webots: samples/devices/console or at `$WEBOTS_HOME/projects/samples/devices/controllers/console` to demonstrate how to use those.
```

Comment:
```suggestion
There is a world example in webots: samples/devices/console or at "[WEBOTS\_HOME/projects/samples/devices/controllers/console](https://github.com/cyberbotics/webots/tree/master/projects/samples/devices/controllers/console)" to demonstrate how to use those.
```

Reply:
Just curious about the double quotes here, do we need them ?

Reply:
We don't *need* them, it is just to be coherent with the other references (see for example: https://www.cyberbotics.com/doc/guide/appearances#asphalt).